YML.md

```diff
@@ -188,21 +209,22 @@ script:
   - cp ../Telegram/Telegram ./usr/bin/telegram-desktop
 ```
 
-In the `ingredients` section, an archive is downloaded and unpacked. Then, in the `script` section, the main executable is copied into place inside the AppDir. 
+In the `ingredients` section, an archive is downloaded and unpacked. Then, in the `script` section, the main executable is copied into place inside the AppDir.
+
 
 ### The script section needs to copy icon and `.desktop` file in place
 
-Since an AppImage may contain more than one executable binary (e.g., helper binaries launched by the main executable) and also may contain multiple `.desktop` files, a clear entry point into the AppImage is required. For this reason, there is the convention that there should be exactly one `$ID.desktop` file and corresponding icon file in the top-level directory of the AppDir.
+Since an AppImage may contain more than one executable binary (e.g. helper binaries launched by the main executable) and also may contain multiple `.desktop` files, a clear entry point into the AppImage is required. For this reason, there is the convention that there should be exactly one `$ID.desktop` file and corresponding icon file in the top-level directory of the AppDir.
```

Comment:
Depends on the flavor of English that is used. In British English, "e.g." is not followed by a comma. In American English, it is _typically_ followed by a comma, although this is not enforced. See also https://en.wiktionary.org/wiki/e.g.